examples/basic/main.tf

```diff
@@ -0,0 +1,38 @@
+
+# ------------------------------------------------------------------------------
+# Module
```

Comment:
add here identification of pull & push [like Module for push subscription and topic ]

Reply:
done sir